docs/tasks.md

```diff
@@ -129,6 +129,12 @@ or container images that you define:
   the configuration file.
 - Each container image runs until completion or until the first failure is
   detected.
+- The CPU, memory, and ephemeral storage resource requests will be set to zero
```

Comment:
Any thoughts on a better place to put this documentation or a better way to phrase it would be welcome!